src/Language/Fixpoint/Graph/Deps.hs

```diff
@@ -553,7 +555,7 @@ instance PTable Stats where
     , ("KVars NonLin"     , pprint stSetKVNonLin)
     ]
 
-graphStats :: F.TaggedC c a => Config -> F.GInfo c a -> Stats
+graphStats :: (Fixpoint s, Ord s, Show s, Eq s, Hashable s) => F.TaggedC c s a => Config -> F.GInfo c s a -> Stats s
```

Comment:
```suggestion
graphStats :: (Fixpoint s, Ord s, Show s, Eq s, Hashable s, F.TaggedC c s a) => Config -> F.GInfo c s a -> Stats s
```

Reply:
I didn't realize => => would typecheck...
I wrote an emacs lisp function which semi-automatically fills in the type constraints, which is likely the reason why this happens.
I'll do a global grep to make sure the => => pattern doesn't occur

Reply:
I rewrote all the lines matching =>.*=> into a single =>. I didn't check any multiline case (much trickier to match with regex) but it should never arise as my automation script operates on a per-line basis.